.github/containerscan/allowedlist.yaml

```diff
@@ -1,6 +1,11 @@
 general:
   vulnerabilities:
     #besu-all-in-one
+    - CVE-2022-37734
+    - CVE-2022-25857
 
-    -CVE-2022-37734
-    -CVE-2022-25857
+    #cactus-example-supply-chain-app
+    - CVE-2022-24434
+    - CVE-2022-24999 #express
+    - CVE-2022-24999 #qs
+    - CVE-2022-2421
```

Comment:
@aldousalvarez Let's just leave these out for now, as-in, revert the changes in allowedlist.yaml 'coz I think we can find some way to fix those vulnerabilities and in the meantime they shouldn't be masked unless absolutely necessary.
Other than this, it LGTM

Reply:
Okay Thank you, noted on that one. Already fixed it.